packages/app/src/components/OnboardingModal.tsx

```diff
@@ -100,26 +100,26 @@ export default function OnboardingModal({
                     password: '',
                   },
                 });
-                const metricsSource = await createSourceMutation.mutateAsync({
+                const logSource = await createSourceMutation.mutateAsync({
```

Comment:
Reorder the source creation calls. The dropdown should show 'Demo Logs' -> 'Demo Traces' -> 'Demo Metrics' -> 'Demo Sessions'